support/cas-server-support-fortress/src/test/resources/fortress.properties

```diff
@@ -0,0 +1,5 @@
+http.user=fortress-super-user
```

Comment:
This sample could perhaps be added to the docs as a decent starting point for the file outline?